qiita_pet/handlers/study_handlers/ebi_handlers.py

```diff
@@ -39,14 +37,26 @@ def display_template(self, preprocessed_data_id, msg, msg_level):
                 raise HTTPError(403, "No permissions of admin, "
                                      "get/EBISubmitHandler: %s!" % user.id)
 
-        prep_template = PrepTemplate(preprocessed_data.prep_template)
-        sample_template = SampleTemplate(preprocessed_data.study)
-        study = Study(preprocessed_data.study)
+        prep_templates = preprocessed_data.prep_templates
+        allow_submission = len(prep_templates) == 1
+        msg_list = ["Submission to EBI disabled:"]
+        if not allow_submission:
+            msg_list.append(
+                "Only artifacts with a single prep template can be submitted")
+        # If allow submission is already false, we technically don't need to
+        # do the following work. However, there is currently no clean way to
```

Comment:
Minor, but you can skip this currently here as you are referring to the *current* structure in the next sentence.